source/deprecations/v1.x.html.md

```diff
@@ -387,3 +387,37 @@ After:
 
 ```handlebars
 {{#with foo as |bar|}}
+```
+
+#### Deprecate using the same function as getter and setter in Computed Properties
+
+Now that we have the new computed property syntax ([RFC](https://github.com/stefanpenner/rfcs/blob/read-only-cp/0000-read-only-cp-syntax.md))
```

Comment:
Final location is: https://github.com/emberjs/rfcs/blob/master/active/0001-improved-cp-syntax.md